app/components/join-steps/step-three.js

```diff
@@ -26,12 +27,16 @@ export default class StepThreeComponent extends Component {
   }
 
   @action inputHandler(e) {
+    console.log('I am ruuning');
```

Comment:
can we please remove the `console.log` if not necessary ?

Reply:
removed✅